src/fosslight_source/run_scanoss.py

```
@@ -69,22 +70,47 @@ def run_scanoss_py(path_to_scan: str, output_path: str = "", format: list = [],
             scanoss_settings=scanoss_settings,
-            timeout=timeout
+            timeout=timeout,
+            retry=0
         )
-        output_buffer = io.StringIO()
         with contextlib.redirect_stdout(output_buffer), contextlib.redirect_stderr(output_buffer):
             scanner.scan_folder_with_options(scan_dir=path_to_scan)
-        captured_output = output_buffer.getvalue()
-        api_limit_exceed = "due to service limits being exceeded" in captured_output
-        timeout_occurred = "The SCANOSS API request timed out" in captured_output
+    except Exception as error:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)